client/app/src/accounts/AccountService.js

```diff
@@ -282,6 +282,19 @@
       });
       return deferred.promise;
     };
+    
+    function getAllRegisteredDelegates() {
+      var deferred = $q.defer();
+      networkService.getFromPeer("/api/delegates").then(function (resp) {
```

Comment:
Oh, I thought it returns all the registered delegates, hence the function name :). My mistake.